docs/guides/BasicUsage.md

```
@@ -93,13 +104,17 @@ async with ctx.scope(  # prepare the new context
     print("RESULT GPT 3.5 | temperature 0.7:", result)
 ```
 
-    RESULT GPT 3.5 | temperature 0.4: Violets are blue.
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)